Resources/Prototypes/Entities/Structures/Wallmounts/intercom.yml

```diff
@@ -220,7 +220,15 @@
   id: IntercomCommand
   parent: Intercom
```

Comment:
make a new parent for reinforced intercoms

Reply:
Thanks Emo, let me know if I did that correctly.